packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -383,6 +383,10 @@ class AndroidDevice extends Device {
       // activity name from the .apk.
       package = await AndroidApk.fromAndroidProject(project.android);
     }
+    // There was a failure parsing the android project information.
+    if (package == null) {
+      throwToolExit('Error in AndroidManifest.');
```

Comment:
What is a user supposed to do in this case?  Is it that the AndroidManifest can't be found, or that it has a syntax error, or something else?

Reply:
Removed this sentence, we'll get the error logging already this just ensures we exit cleanly